CHANGELOG.md

```diff
@@ -2,4 +2,6 @@
 
 We use this CHANGELOG to document breaking changes, new features, bug fixes,
 and config value changes that may affect both the usage of the workflows and
-the outputs of the workflows.
+the outputs of the workflows. See [the Nextstrain CLI
+changelog](https://github.com/nextstrain/cli/blob/master/CHANGES.md#readme)
+for an example of formatting.
```

Comment:
_non-blocking_

While I'm very partial to the CLI's changelog, the [ncov repo's changelog](https://github.com/nextstrain/ncov/blob/HEAD/docs/src/reference/change_log.md) may be more appropriate for a workflow?

Separately, instead of linking to `master` or `main`, we prefer to link to `HEAD` or `@` or `-` as that will be correct regardless of default branch name changing in the future. e.g. <https://github.com/nextstrain/cli/blob/HEAD/CHANGES.md#readme>. Both `HEAD` and `@` are preserved, while `-` issues a redirect to the actual branch name.